include/tvm/runtime/container/shape_tuple.h

```diff
@@ -42,6 +43,9 @@ class ShapeTupleObj : public Object {
   /*! \brief The size of the shape tuple object. */
   uint64_t size;
 
+  /*! \brief Get number of elements in the shape */
+  index_type Numel() const;
```

Comment:
trying to dig a bit about related APIs. How about Product? the product of the shape values

Reply:
numel makes sense for an array api. e.g. array.numel. Just not sure if it is right to use it for shape.

Reply:
How about NumElements()?

Reply:
OK. I don't have strong opinion to use `Product`. It's just `numel` is a well-known convention, but `Product` is a newly-invented word with ambiguity, but I don't have strong opinion.